modules/authentication/proc-provisioning-users-from-github-to-the-software-catalog.adoc

```diff
@@ -0,0 +1,76 @@
+[id="provisioning-users-from-github-to-the-software-catalog"]
+= Provisioning users from GitHub to the software catalog
+
+To authenticate users, {product} requires their presence in the software catalog.
+Consider configuring {product-short} to provision users from GitHub to the software catalog on schedule, rather than provisioning the users manually.
+
+.Prerequisites
+* You xref:enabling-authentication-with-github[enabled authentication with GitHub], including the following secrets:
+** `GITHUB_HOST_DOMAIN`
+** `GITHUB_ORGANIZATION`
+
+.Procedure
+
+* To enable GitHub member discovery, edit your custom {product-short} ConfigMap, such as `app-config-rhdh`, and add the following lines to the `app-config-rhdh.yaml` content:
++
+--
+[id=githubProviderId]
+.`app-config.yaml` fragment with mandatory `github` fields
+[source,yaml]
+----
+dangerouslyAllowSignInWithoutUserInCatalog: false
```

Comment:
@themr0c Consider adding number tags and descriptions for the parameters in the fragment.